SWIG/vectors.i

```diff
@@ -34,6 +35,7 @@ namespace std {
     %template(DoubleVector) vector<double>;
     %template(StrVector) vector<std::string>;
     %template(BoolVector) vector<bool>;
+    %template(SizeVector) vector<Size>;
```

Comment:
I think this (and the change to `Size` above, which kind of worried me) are no longer needed now.